src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/ClaspLoggingBridge.cs

```diff
@@ -0,0 +1,114 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNetCore.Razor.Telemetry;
+using Microsoft.CodeAnalysis.Razor.Logging;
+using Microsoft.CommonLanguageServerProtocol.Framework;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.AspNetCore.Razor.LanguageServer;
+
+/// <summary>
+/// Providers a bridge from CLaSP, which uses ILspLogger, to our logging infrastructure, which uses ILogger
+/// </summary>
+internal class ClaspLoggingBridge : ILspLogger
```

Comment:
This is actually a rename of `LoggerAdapter`

Reply:
You mentioned the Clasp logic is in Roslyn. Which other repositories (aside from Razor) also make use of ILspLogger.

Reply:
Roslyn, Razor, Web Tools and I think Bicep all use CLaSP. There might be one other I am forgetting